src/cards/rachelWCard/rachelWCard.scss

```diff
@@ -5,20 +5,50 @@
     grid-template-rows: auto;
     align-items: center;
 }
+
+.cardImage{
+    background-image: url(./dragonfly.jpg);
+    background-size: 100%;
+    background-position: 20%;
+}
+.cardImage:hover { transform: scale(1.05); }
+
 .card{
     margin: auto;
+    color: white;
```

Comment:
I'm seeing this .card color setting spill over to the other cards in the preview ( thier text became white too), can you try making your .card class uniquely named for example .rachelWCard, then update in the .jsx as well. This would be a good rule to any of the other elements as well :)

Reply:
i would also do a git pull to grab all the latest main changes :)

Reply:
Okay, I fixed the issues!